packages/vite/src/node/plugins/asset.ts

```diff
@@ -43,7 +43,7 @@ const assetCache = new WeakMap<ResolvedConfig, Map<string, string>>()
 // For the manifest, we need to preserve the original file path and isEntry
 // for CSS assets. We keep a map from referenceId to this information.
 export interface GeneratedAssetMeta {
-  originalName: string
+  originalFileName: string
   isEntry?: boolean
 }
 export const generatedAssets = new WeakMap<
```

Comment:
We still need to maintain the internal list to output deduped files in the manifest.